### ISSUE TITLE
Avoid specifying a new window as the target of a link with target="_blank".

### ISSUE THREAD

**Issue by lfrancis** [cultureondemand/iava-report#8]
Avoid specifying a new window as the target of a link with target="_blank".
WCAG 2.0 AAA F22
1 pages

Displaying new windows without warning can be very confusing to non-sighted and mobile users. Some screen readers and mobile devices give very little indication a new tab or window has opened. If you cannot avoid displaying a new window, insert a warning into the link text like the following: "opens in a new window".

https://cultureondemand.github.io/iava-report/
Line 309
